api/server/server.go

```diff
@@ -117,19 +121,25 @@ func (s *Server) ServeApi(protoAddrs []string) error {
 	return nil
 }
 
-type HttpServer struct {
+// HTTPServer contains an instance of http server and the listener.
```

Comment:
let's write `http.Server` and `net.Listener` to have reference to real types in godoc.